docs/swagger.json

```diff
@@ -666,29 +666,6 @@
                 }
             }
         },
-        "/ping": {
```

Comment:
why did we remove the healthcheck endpoint?

Reply:
it was a part of the rmbproxy server. restored now.